model/src/guild/audit_log/change.rs

```diff
@@ -37,7 +38,7 @@ pub struct AffectedRole {
 /// Individual change within an [`AuditLogEntry`].
 ///
 /// [`AuditLogEntry`]: super::AuditLogEntry
-#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
```

Comment:
why not add a `Hash` derive for `GuildCommandPermissions`?

Reply:
Right, I deleted the impls without thinking.